ghost/core/core/server/data/migrations/versions/5.97/2023-10-10-01-02-03-add-signin-urls-permissions.js

```diff
@@ -0,0 +1,8 @@
+const {combineTransactionalMigrations, addPermissionToRole} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
+    addPermissionToRole({
+        permission: 'Read member signin urls',
+        role: 'Admin Integration'
+    })
+);
```

Comment:
```suggestion
module.exports = addPermissionToRole({
    permission: 'Read member signin urls',
    role: 'Admin Integration'
});
```

I don't think we need the `combineTransactionalMigrations` as there is only 1 transactional migration here (`addPermissionToRole` creates a `createTransactionalMigration` under the hood)